newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -269,6 +274,7 @@ export type Preferences = {|
   setShowGetStartedSection: (enabled: boolean) => void,
   setShowEventBasedObjectsEditor: (enabled: boolean) => void,
   getShowEventBasedObjectsEditor: () => boolean,
+  setDefaultWorkspace: (defaultWorkspace: string) => void,
```

Comment:
I would recommend something like:
```suggestion
  setDefaultWorkspaceFolder: (defaultWorkspace: string) => void,
```

or even better:
```suggestion
  setNewProjectsDefaultFolder: (defaultFolder: string) => void,
```